src/xrAICore/Components/problem_solver.h

```diff
@@ -151,4 +151,434 @@ class CProblemSolver
     virtual void clear();
 };
 
-#include "xrAICore/Components/problem_solver_inline.h"
+// #include "xrAICore/Components/problem_solver_inline.h"
```

Comment:
Please, don`t do this. Remove, if not needed. And same things in others files...